OpenArtemis/Views/Subreddits/SubredditFeedView.swift

```diff
@@ -74,10 +75,11 @@ struct SubredditFeedView: View {
                             }
                         }
                     }
-                    
+
                     Rectangle()
                         .fill(Color.clear)
                         .frame(height: 1)
+                        .id(UUID()) // adding this causes onAppear to be called multiple times even if the view didn't leave the screen
```

Comment:
Keep this

Reply:
Is the comment here true, that it is triggering the on appear multiple times?